zombies.js

```diff
@@ -242,12 +386,18 @@
  */
 
 
+
 /**
  * FastZombie Extends Zombie Class
  * -----------------------------
  */
 
-
+class FastZombie extends Zombie{
+  constructor(health, strength, speed){
+    super(health, strength, speed);
+    this.speed = speed;
```

Comment:
speed is part of the super constructor, so I does not need to be set on the child